testsuite/integration-arquillian/servers/app-server/jboss/wildfly/pom.xml

```diff
@@ -30,13 +30,13 @@
     <name>App Server - Wildfly</name>
     
     <properties>
-        <wildfly.version>23.0.2.Final</wildfly.version>
+        <wildfly.server.version>23.0.2.Final</wildfly.server.version>
```

Comment:
Should this be updated to a later version?

Reply:
As discussed with @pskopek, this is about the test suite and could/should IMHO be a follow up issue.